pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools>=69"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "PyFronius"
+version = "0.7.6"
+authors = [
+    { name = "Niels Mündler", email = "n.muendler@web.de" },
+    { name = "Matthias Alphart", email = "farmio@alphart.net" },
+    { name = "Gerrit Beine", email = "mail@gerritbeine.de" },
+]
+description = "Client for Fronius SolarAPI JSON interface"
+license = { text = "MIT" }
+keywords = ["Fronius", "SolarAPI", "photovoltaics", "pv"]
+readme = "README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+    "License :: OSI Approved :: MIT License",
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Object Brokering",
+]
+dependencies = ["aiohttp"]
+requires-python = ">=3.8.0"
```

Comment:
Should we not also specify the latest version that we actively support? I try to keep this in sync with the set of python versions tested by CI

Reply:
We can do this via classifiers if you like. `requires-python` only lets us set a minimum version.